e2e-tests/portal/cypress/integration/journeys/dashboard/dashboard-header.spec.js

```diff
@@ -0,0 +1,76 @@
+const relative = require('../../relativeURL');
+
+import page from '../../pages/header';
+
+const { dashboard } = require('../../pages');
+
+const mockUsers = require('../../../fixtures/mockUsers');
+
+const BANK1_MAKER1 = mockUsers.find((user) => (user.roles.includes('maker')));
+
+context('Portal GOVUK header displays correctly', () => {
```

Comment:
File is called dashboard-header, but it's not specific to the dashboard, can we move outside of dashboard directory?

Reply:
isn't it logging into the dashboard?

Reply:
done